modules/helm-initializer/main.tf

```diff
@@ -39,7 +39,7 @@ echo '${data.template_file.tiller_rbac.rendered}' | kubectl apply -f - \
 --tiller-tls-cert=${local_file.tiller_cert.filename} \
 --tiller-tls-key=${local_file.tiller_key.filename} \
 --override 'spec.template.spec.containers[0].command'='{/tiller,--storage=secret}' \
-&& sleep 30 \
+&& sleep ${var.tiller_wait_period} \
```

Comment:
Nice little fix for Tiller timeouts, that I am using in https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/helm-initializer/main.tf#L19